Simplified/Reader2/BusinessLogic/NYPLLastReadPositionPoster.swift

```diff
@@ -56,16 +56,22 @@ class NYPLLastReadPositionPoster {
     guard let location = NYPLBookLocation(locator: locator) else {
       return
     }
+
+    // save location locally, so that it can be later be saved on disk
     bookRegistryProvider.setLocation(location, forIdentifier: book.identifier)
 
-    let selectorValue = NYPLBookmarkFactory
-      .makeLocatorString(chapterHref: locator.href,
-                         chapterProgression: Float(chapterProgress))
-    postReadPosition(selectorValue: selectorValue)
+    // attempt to store location on server
+    if NYPLAnnotations.syncIsPossibleAndPermitted() {
+      let selectorValue = NYPLBookmarkFactory
+        .makeLocatorString(chapterHref: locator.href,
+                           chapterProgression: Float(chapterProgress))
+
+      postReadPosition(selectorValue: selectorValue)
+    }
   }
 
-  /// Deprecated
```

Comment:
@ErnestFan I removed the deprecation notice because we are still using it... unless you remember why we put it there?

Reply:
I don't really remember, but here is the commit ([link](https://github.com/NYPL-Simplified/Simplified-iOS/commit/3b12829c5cea3c3a67ab1f23eaf65bcc89f0f2bc#))
I guess you added the `deprecated` because it was posting hardcoded CFI, and then you refactored it to use the `locationString` and forgot to remove the `deprecated`?
Anyways, yes removing it is fine if we are still using it